template/gradle/libs.versions.toml

```diff
@@ -7,10 +7,12 @@ gradle-android = "7.3.0"
 gradle-versions = "0.43.0"
 
 compose = "1.3.0-beta02"
+compose-material = "1.0.0-rc01"
```

Comment:
Wasn't it stable?

Reply:
yes it is :) 
https://mvnrepository.com/artifact/androidx.compose.material3/material3

on the [website](https://developer.android.com/jetpack/androidx/releases/compose-material3) it does now show, but I double checked